tools/report.sh

```diff
@@ -107,44 +106,43 @@ resources_to_fetch=(
 	"roles"
 	"rolebindings"
 	"networkpolicies"
-	"routes"
 	"pods"
 	"persistentvolumeclaims"
+	"routes"
 	)
 
 if [ "$secrets" = "off" ]; then
-  resources_to_fetch=( "${resources_to_fetch[@]/secrets}" )
+  resources_to_fetch=("${resources_to_fetch[@]/secrets}")
 fi
 
 get_masked_secrets() {
-	mkdir -p $direct/reports/"$1"
+	mkdir -p $direct/reports/$1
```

Comment:
Doublequoting is normally recommended everywhere. What is your reason for removing it? (not just here but in many other places) The script was written before we used shellcheck and does not pass currently. But IIRC this would be one of the things it would complain about. If we one day get to use it here as well, we might need to revert all these back?

Reply:
So why not using quotes also for $direct? For me it's more about the context in which it is used that matters. Anyway, I understand the project standard, so I'll run shellcheck on it and fix any warning/error for future proof.

Reply:
Well, as I said, it does not pass shellcheck currently (even before your PR I mean). And you don't have to follow it to fix all the points. But we should not move in the oposite direction either.

Reply:
Yeah, I agree.